code/modules/vehicles/mecha/mech_fabricator.dm

```diff
@@ -191,6 +194,13 @@
 		else if(ispath(built_item, /obj/item/borg_restart_board))
 			sub_category += "All Cyborgs" //Otherwise the restart board shows in the "parts" category, which seems dumb
 
+		// SKYRAT EDIT START - ipc_printing
+
+		else if(findtext(built_item, "robot_ipc") || findtext(built_item,"power_cord"))
```

Comment:
I got a feeling that's probably not the best way to do this, have you tried using `ispath()` like the other ones?